src/Raven.Studio/wwwroot/App/views/database/patch/patch.html

```diff
@@ -111,98 +111,212 @@ <h2 class="on-base-background">Patch</h2>
                                 </div>
+                                <ul data-bind="foreach: actions.putDocument">
+                                    <li class="margin-left margin-left-lg row">
+                                        <small class="col-sm-2 margin-left margin-left-lg"><label>ID:</label><span class="margin-left tab-data" data-bind="text: $data.Id"></span></small>
+                                        <small class="col-sm-2">
+                                            <label>Put Document:</label>
+                                            <span class="margin-left tab-data"  data-bind="css: { 'hide' : $parent.showDocumentsInModified }">{ ... }</span>
+                                        </small>
+                                        <pre data-bind="html: $parent.formatAsJson($data.Data), css: { 'show' : $parent.showDocumentsInModified }"></pre>
+                                    </li>
+                                </ul>
                             </div>
-                            <div class="flex-grow scroll" data-bind="foreach: actions.putDocument">
-                                <div class="put-item">
-                                    <div class="put-heading" data-bind="text: Id"></div>
-                                    <div class="put-placeholder">{ ... }</div>
-                                    <pre data-bind="html: $parent.formatAsJson(Data)"></pre>
+                            <div data-bind="visible: actions.incrementCounter().length">
+                                <hr>
+                                <h3><i class="icon-new-counter"></i><span>Counters</span></h3>
+                                <ul data-bind="foreach: actions.incrementCounter">
+                                    <li class="margin-left margin-left-lg row">
+                                        <small class="col-sm-2 margin-left margin-left-lg"><label>Name:</label><span class="margin-left tab-data" data-bind="text: $data.Name"></span></small>
+                                        <small class="col-sm-2"><label>Value Before:</label><span class="margin-left tab-data" data-bind="text: $data.Created ? '&lt;None&gt;' : $data.OldValue"></span></small>
+                                        <small class="col-sm-2"><label>Value After:</label><span class="margin-left tab-data" data-bind="text: $data.NewValue"></span></small>
+                                        <small class="col-sm-2"><span class="tab-data" data-bind="text: $data.Created ? '&lt;Counter created&gt;' : ''"></span></small>
+                                    </li>
+                                </ul>
+                            </div>
+                            <div data-bind="visible: actions.appendTimeseries().length">
+                                <hr>
+                                <div class="flex-horizontal">
+                                    <h3><i class="icon-document"></i><span>Time Series</span></h3>
```

Comment:
I always wonder what's proper casing for this word:
Timeseries
TimeSeries
Time Series? 
;-)

Reply:
in your code sometimes you have timeseries and sometimes timeSeries.

Reply:
I will unify:

**for text/docs:** 
     *   Time Series  => as a title/heading
     *   time series  => inside plain text
     *  Time series => beginning of a sentence

**for ts code:** 
     *  timeSeries => imho